### PR TITLE
Rendersettings

### PR DESCRIPTION
**Pull request by haggi**
Remove "Appleseed" from the render globals tabs. Restructure the output options in the translator globals tab.

### CODE REVIEW

File: module/scripts/appleseed/initialize.py [hunk 1]
```diff
@@ -413,36 +413,45 @@ def AppleseedRendererUpdateTab(self, dummy=None):
...
 
-    def xmlFileBrowse(self, args=None):
-        filename = pm.fileDialog2(fileMode=0, caption="XML Export File Name")
-        if len(filename) > 0:
-            self.rendererTabUiDict['xml']['xmlFile'].setText(filename[0])
+    def outputFileBrowse(self, args=None):
+        startingDir = path.path(self.renderGlobalsNode.exportSceneFileName.get()).dirname()
+        filename = pm.fileDialog2(fileMode=0, caption="Output File Name", startingDirectory=startingDir)
+        if filename:
+            if len(filename) > 0:
```
Comment (dictoon) [inline module/scripts/appleseed/initialize.py]: Isn't `if filename:` sufficient?

File: src/appleseedrenderer.cpp [hunk 2]
```diff
@@ -178,6 +178,16 @@ void AppleseedRenderer::render()
                     tileCallbackFac.get()));
         }
 
+        MFnDependencyNode renderGlobalsFn(getRenderGlobalsNode());
+        int exportMode = getEnumInt("exportMode", renderGlobalsFn);
```
Comment (dictoon) [inline src/appleseedrenderer.cpp]: Make const.

File: src/appleseedrenderer.cpp [hunk 3]
```diff
@@ -178,6 +178,16 @@ void AppleseedRenderer::render()
...
+        if (exportMode > 0)
+        {
+            MString outputFile = getStringAttr("exportSceneFileName", renderGlobalsFn, "");
```
Comment (dictoon) [inline src/appleseedrenderer.cpp]: Make const.

File: module/scripts/appleseed/initialize.py (hunk 1 above)
Comment (haggi) [inline module/scripts/appleseed/initialize.py]: Indeed, it is.

**Comment by dictoon** [comment]
Thanks!

**Pull request merged**